src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/entities/TaskHistoryResource.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.time.OffsetDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ToString
+@Getter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Entity(name = "taskHistory")
+@SuppressWarnings({"PMD.TooManyFields"})
+public class TaskHistoryResource {
```

Comment:
Not important but could this be a Java 17 Record?  There's some customisation required for equalandhashcode so may be more trouble than it's worth.